modules/exploits/windows/http/sharepoint_ssi_viewstate.rb

```diff
@@ -92,7 +92,8 @@ def initialize(info = {})
       OptString.new('VALIDATION_KEY', [false, 'ViewState validation key']),
       # "Promote" these advanced options so we don't have to pass around our own
       OptString.new('HttpUsername', [false, 'SharePoint username']),
-      OptString.new('HttpPassword', [false, 'SharePoint password'])
+      OptString.new('HttpPassword', [false, 'SharePoint password']),
+      OptString.new('Cookie', [false, 'Cookie'])
```

Comment:
Just a little more descriptive.

```suggestion
      OptString.new('Cookie', [false, 'SharePoint cookie if you have one'])
```

If you can, please move this option above the comment about promoting advanced options. Make sure to remove the trailing comma. Thanks!